src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaConfigCommon.java

```diff
@@ -390,18 +391,35 @@ public KafkaConfigCommon(CVService cvService,
                                 siste14aVedtakService::behandleKafkaRecord
                         );
 
+        KafkaConsumerClientBuilder.TopicConfig<String, CVMelding> cvHjemmelRewindConfig = new KafkaConsumerClientBuilder.TopicConfig<String, CVMelding>()
+                .withLogging()
+                .withMetrics(prometheusMeterRegistry)
+                .withStoreOnFailure(consumerRepository)
+                .withConsumerConfig(
+                        Topic.CV_TOPIC.topicName,
+                        Deserializers.stringDeserializer(),
+                        Deserializers.jsonDeserializer(CVMelding.class),
+                        cvService::behandleKafkaMeldingCVHjemmel
+                );
+
         consumerClientAivenSiste14a = KafkaConsumerClientBuilder.builder()
                 .withProperties(aivenDefaultConsumerProperties(CLIENT_ID_CONFIG))
                 .withTopicConfig(siste14aTopicConfig)
                 .withToggle(() -> unleashService.isEnabled(KAFKA_SISTE_14A_STOP) || kafkaAivenUnleash.get())
                 .build();
 
+        consumerClientCvHjemmelRewind = KafkaConsumerClientBuilder.builder()
+                .withProperties(aivenDefaultConsumerProperties(CLIENT_ID_CONFIG + "-rewind"))
```

Comment:
Bare for læring: hvorfor lager man en ny "id" for applikasjonen her? :) er det for å få nytt offset osv?

Reply:
yep yep, ny consumer skal begynne fra eldest offset